packages/ra-auth-cognito/src/ErrorMFARequired.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Error thrown when a user is required to enter a MFA code from SMS.
+ * Unsupported.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** If MFA code from SMS is Unsupported, is this error still useful? ‏